com.rosegardenmusic.rosegarden.json

```diff
@@ -62,17 +62,20 @@
       "sources": [
         {
           "type": "archive",
-          "url": "https://downloads.sourceforge.net/project/rosegarden/rosegarden/23.12/rosegarden-23.12.tar.xz",
-          "sha256": "3f57e66460044efea082379c9706612fe135b6e81f16cbf0bebe859174fe9332",
-          "x-checked-data": {
+          "url": "https://downloads.sourceforge.net/project/rosegarden/rosegarden/24.06/rosegarden-24.06.tar.xz",
+          "sha256": "866cd4297c128a68208edd28a37808f278f630219533fff32441ad6647e09c27",
+          "x-checker-data": {
               "type": "anitya",
               "project-id": 8939,
               "url-template": "https://downloads.sourceforge.net/project/rosegarden/rosegarden/$version/rosegarden-$version.tar.xz"
           }
         },
         {
           "type": "patch",
-          "path": "rosegarden-mime-icon.patch"
```

Comment:
why do you remove this? One doesn't simply remove a patch.

Reply:
I thought, since you implied [in your previous comment](https://github.com/flathub/com.rosegardenmusic.rosegarden/pull/15#issuecomment-2156173760) that you were meant to have dropped the MIME icon patch yourself, it didn't need to be there anymore. [The test build that just ran without this patch has been successfully built](https://buildbot.flathub.org/#/builders/6/builds/126870), but if you still need me to bring the patch back, I can do that, no problem.

Reply:
I never said to remove it.

Reply:
I said

> I'm supposed to drop it now that I fixed https://github.com/flatpak/flatpak-builder/issues/1 .

That doesn't mean delete. If you read the issue I referenced, it tell that something else need to be done to achieve the same outcome.

Reply:
I case it is still unclear. The patch needs to be back in the build.

Reply:
Ah, I misread what you said, then. Sorry.

Reply:
> I case it is still unclear. The patch needs to be back in the build.

[I brought the patch back just now](https://github.com/flathub/com.rosegardenmusic.rosegarden/pull/15/commits/1e7ba5bac36d7f65f8f9bb6fa019675640a6361c). Sorry for all the confusion on my end. If I need to anything else before this PR gets merged, please tell me. I'm not bringing the patch back to the upstream repository since, as you said earlier, it's only meant for this Flatpak.

Reply:
In fact, the 2 most recent commits I pushed today, that basically took out the patch and brought it back again, don't need to be in my fork anymore. Or anywhere, for that matter. [Once the currently running test build succeeds](https://buildbot.flathub.org/#/builders/6/builds/126878), I'll force push them out of here and hopefully this should be good to go.

Reply:
I just force pushed the two offending commits out of here. [A test build stemming from said force push has just commenced](https://buildbot.flathub.org/#/builders/6/builds/126880). Once it succeeds, this PR _should_ be good to go, but if anything else needs changing, please let me know.